Assets/Scripts/Enums/BodyPartType.cs

```diff
@@ -0,0 +1,18 @@
+namespace Enums
+{
+    public enum BodyPartType
+    {
+        EyeLeft,
+        EyeRight,
+        BicepsLeft,
+        BicepsRight,
```

Comment:
It's should be called `UpperArm`? Or just `Arm` since there is no `LowerArm`

Reply:
This is based on the model names of the body parts. The armatures are named `UpperArm_L`, `LowerArm_L` but the model is simply `BicepsLeft`.

I am all for unifying the same across everywhere. Someone needs to make a decision which variant to use.

Reply:
I'll bring this up with a modeller

Reply:
mine is not named BicepsLeft
![image](https://user-images.githubusercontent.com/46575243/72223127-8a28c080-354a-11ea-816e-da9ae3ca6bca.png)